src/app/pages/account/edit/[username].vue

```diff
@@ -1,56 +1,181 @@
+              {{ account?.description }}
+            </TypographyBodyMedium>
+          </div>
+          <div
+            v-if="isEditing"
+            class="flex flex-col items-end gap-2 text-right"
+          >
+            <TypographySubtitleSmall
+              class="text-(--semantic-base-text-secondary)"
+            >
+              {{
+                t('maxCharacters', { count: editableDescription?.length || 0 })
+              }}
+            </TypographySubtitleSmall>
+            <ButtonColored
+              variant="secondary"
+              :aria-label="t('saveChanges')"
+              @click="saveDescription"
+            >
+              <TypographyLabel>
+                {{ t('saveChanges') }}
+              </TypographyLabel>
+            </ButtonColored>
+          </div>
+        </div>
+        <CardButton
+          class="border-(--warning-strong) bg-(--warning-weak) text-(--warning-text)"
+          :title="t('resetPassword')"
+          :to="
+            localePath({
+              name: 'account-password-reset-request',
+            })
+          "
+        >
+          <AppIconVerifiedUser />
+          <template #iconSecondary>
+            <AppIconRestartAlt />
+          </template>
+        </CardButton>
+        <div class="flex justify-center">
+          <TypographySubtitleMedium
+            class="cursor-pointer text-(--semantic-critic-text) underline underline-offset-2"
+            @click="openDeleteDrawer"
+          >
+            {{ t('deleteAccount') }}
+          </TypographySubtitleMedium>
+        </div>
+        <AccountDeleteDrawer
+          v-model:is-open="isDeleteDrawerOpen"
+          :account-id="account?.id"
         />
-      </section>
-    </div>
+      </div>
+    </LayoutPage>
   </Loader>
 </template>
 
 <script setup lang="ts">
-import { useAccountDeleteMutation } from '~~/gql/documents/mutations/account/accountDelete'
 import { useProfilePictureSetMutation } from '~~/gql/documents/mutations/profilePicture/profilePictureSet'
 import { useAccountByUsernameQuery } from '~~/gql/documents/queries/account/accountByUsername'
 import { getAccountItem } from '~~/gql/documents/fragments/accountItem'
+import { useAccountDescriptionUpdateMutation } from '~~/gql/documents/mutations/account/accountDescriptionUpdate'
+import { getProfilePictureItem } from '~~/gql/documents/fragments/profilePictureItem'
+import { getUploadItem } from '~~/gql/documents/fragments/uploadItem'
+import { useProfilePictureByAccountIdQuery } from '~~/gql/documents/queries/profilePicture/profilePictureByAccountId'
+import { useDeleteUploadByIdMutation } from '~~/gql/documents/mutations/upload/uploadDeleteById'
+
+definePageMeta({
+  layout: 'default-no-header',
+})
 
 const store = useStore()
-const { signOut } = await useSignOut()
+const localePath = useLocalePath()
 const { t } = useI18n()
 
+const isEditing = ref(false)
+const editableDescription = ref('')
+const isDeleteDrawerOpen = ref(false)
```

Comment:
Generally we initialize with `undefined`, setting values later

```suggestion
const isEditing = ref<boolean>()
const editableDescription = ref<string>()
const isDeleteDrawerOpen = ref<boolean>()
```